client/src/components/pages/MembersPage/MembersPage.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import styled from 'styled-components'
+import CreateForm from './components/CreateForm/CreateForm'
+import { ControlBar, ControlBarDisabled } from './components/ControlBar'
+import {
+  Visualisation,
+  VisualisationDisabled,
+} from './components/MembersVisualisation'
+
+import useForm from '../../../hooks/useForm'
+
+function MembersPage({
+  members,
+  setMembers,
+  bands,
+  setBands,
+  getMembers,
+  postMember,
+  patchMember,
+  deleteMember,
+}) {
+  const {
+    formIsVisible,
+    toggleForm,
+    newMembersInstruments,
+    addNewMembersInstrument,
+    removeNewMembersInstrument,
+    resetNewMembersInstruments,
+  } = useForm()
+
+  return (
+    <Grid>
+      {formIsVisible ? (
```

Comment:
Maybe separate isVisible from the logic, that handles the form input? This way you could put useForm inside the actual form instead of passing everything down.